migration/README.md

```diff
@@ -0,0 +1,30 @@
+# Migration
+
+Module containing OpenRewrite recipes for migrating Axon Framework applications.
```

Comment:
I would like the description to link to the standard as well. This would proof helpful for users stumbling on this module, without knowing what OpenRewrite is.

Reply:
Sure; `OpenRewrite` in the text now links to the documentation; Hope that's what you meant.

Reply:
That's exactly what I meant. Thanks!